exchanges/bitmex/bitmex.go

```diff
@@ -92,10 +93,10 @@ const (
 	bitmexEndpointUserWalletSummary     = "/user/walletSummary"
 	bitmexEndpointUserRequestWithdraw   = "/user/requestWithdrawal"
 
-	// Rate limits - 150 requests per 5 minutes
-	bitmexUnauthRate = 30
-	// 300 requests per 5 minutes
-	bitmexAuthRate = 40
+	// rate limits
+	bitmexRateInterval = time.Minute
+	bitmexUnauthRate   = 30
+	bitmexAuthRate     = 60
```

Comment:
batch orders have a limit of (1/10)*bitmexAuthRate can u add that one pls

https://www.bitmex.com/app/restAPI#Limits